common/persistence/serialization/thrift_mapper.go

```diff
@@ -690,6 +690,8 @@ func replicationTaskInfoFromThrift(info *sqlblobs.ReplicationTaskInfo) *Replicat
 	}
 }
 
+var zeroTimeNanos = time.Time{}.UnixNano()
```

Comment:
nit: can the Time.IsZero() function be used instead: https://golang.org/src/time/time.go?s=15971:15998#L416

Reply:
Not in this case. IsZero can tell whether time.Time isself is zero. But we need to check corresponding unix nanos value, that got persisted. And it gets this weird value, due to some internal overflow.